src/pages/index.tsx

```diff
@@ -209,5 +62,32 @@ export const IndexPageQuery = graphql`
         }
       }
     }
+
+    allMarkdownRemark(
```

Comment:
we query for the blog posts but then pass the into the actual page component. This keeps things cleaned up 👍